packages/dcos-history/extra/history/statebuffer.py

```diff
@@ -83,7 +83,7 @@ def update_and_ff(f_path, ff_end):
                 # Set timestamp to None for memory-only buffer updates
                 with open(f_path, 'r') as fh:
                     self._update_buffer(fh.read())
-                while (ff_end-self.update_period) >= self.next_update:
+                while (ff_end - self.update_period) >= self.next_update:
```

Comment:
That is the one my editor kept saying and I was wondering _why_ flake8 wasn't finding it / erroring